### ISSUE TITLE
split "Fast Rise/Drop" into two separate alarms.

### ISSUE THREAD

**Issue by Joerg-Schoemer** [nightscout/nightguard#189]
it would be nice to have separate settings for rise and drop.
e.g. when FPU are coming up after a meal: I don't want to get the alarms by dropping (normally rapidly) only when rises.

**Comment by dhermanns** [comment]
Yes - this could be helpful. I added this as an enhancement. But my time is very limited right now. Maybe someone else jumps in...
